dev-tools/packer/platforms/darwin/pkg/templates/launchd-daemon.plist.j2

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>Label</key>
+        <string>{{.identifier_base}}.{{.beat_name}}</string>
+	<key>ProgramArguments</key>
+	<array>
+            <string>{{.install_path}}/{{.beat_vendor}}/{{.beat_name}}/bin/{{.beat_name}}</string>
+            <string>-c</string>
+            <string>/etc/{{.beat_name}}/{{.beat_name}}.yml</string>
+            <string>--path.home</string>
+            <string>{{.install_path}}/{{.beat_vendor}}/{{.beat_name}}</string>
+            <string>--path.config</string>
+            <string>/etc/{{.beat_name}}</string>
+            <string>--path.data</string>
+            <string>{{.install_path}}/{{.beat_vendor}}/{{.beat_name}}</string>
```

Comment:
Would it be OK to set path.data to `/var/lib/{{.beat_name}}` similar to Linux?

https://github.com/elastic/beats/blob/545938e899c9f085a9b3b7d1d8dd119c139c7a90/dev-tools/packer/platforms/centos/beatname.sh.j2#L9

Reply:
Done